src/app/modules/dashboard/modules/day-plan/components/day-plan-stats/day-plan-stats.component.ts

```diff
@@ -1,15 +1,69 @@
 import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import {Summary} from '../../../../../../api/models/summary';
+import {Store} from '@ngrx/store';
+import {AppState} from '../../../../../../app.recuder';
+import * as fromDayPlan from '../../day-plan.reducer';
+import * as DayPlanActions from '../../day-plan.actions';
+import {Observable} from 'rxjs';
 
 @Component({
   selector: 'diet-day-plan-stats',
   template: `
-      <p>
-          day-plan-stats works!
-      </p>
+    <div class="day-plan-summary-wrapper" [ngClass]="{'day-plan-summary-wrapper-hidden':(shouldShowStats$ | async)==false }">
+        <button class="summary-arrow"
+                [ngClass]="{'rotated-summary-arrow': (shouldShowStats$ | async)==true}"
+                (click)="onArrowClick()"
+                title="{{'DAY_PLAN.TOGGLE_SUMMARY' | translate}}">
+        </button>
+        <div class="summary-element">
```

Comment:
z tego trzeba będzie wydzielić komponent

Reply:
To jest to co mi sie sypało jak probowalem. Zerknij jakbyś mógł